crates/turborepo-ui/src/tui/table.rs

```diff
@@ -124,21 +124,24 @@ impl<'a> StatefulWidget for &'a TaskTable<'a> {
         .column_spacing(0)
         .block(Block::new().borders(Borders::RIGHT))
         .header(
-            vec![format!("Tasks\n{bar}"), " \n─".to_owned()]
-                .into_iter()
-                .map(Cell::from)
-                .collect::<Row>()
-                .height(2),
+            vec![Text::styled(
+                "Tasks\n",
```

Comment:
I don't think this really matters since the `height(1)` is set, but may as well make the cell text a single line
```suggestion
                "Tasks",
```